judge/default/test_script.sh

```diff
@@ -9,18 +9,16 @@ NA=5
 # If the output "passes" the requirements, then
 # return $PASS, otherwise return $FAIL
 # Any technical glitch returns $NA
-validate_submission_output() {
-  diff $1 $2 > /dev/null
+diff -b -Z $1 $2 > /dev/null
```

Comment:
The whole file can be renamed as `diffnormal` or sth.

Reply:
Your call. I was thinking for consistency of naming (since it is anyway defualt), we stick to test_script.sh

Reply:
What we can do is the following - place `difffloat.py` in a folder named `examples` inside `default` i.e., `default/examples/difffloat.py`.

Reply:
Good idea. I'll move it. I'll leave this `test_script.sh` here.